apps/system/js/sync_manager.js

```diff
@@ -218,7 +219,11 @@
         case 'disable':
         case 'sync':
           try {
-            Service.request('SyncStateMachine:' + request.name).catch(e => {
+            Service.request('SyncStateMachine:' + request.name).then(() => {
+              return LazyLoader.load('/shared/js/sync/telemetry.js');
```

Comment:
We are trying to record three user actions: enable, disable and manual sync. From the sync app we don't know when these actions happen. We only know that a sync was requested, but we don't know if it was a manual request or the sync manager timer firing.